joomla/joomla-cms
# [4.0] awesomplete accessibility fix

We use the library awesomeplete in a few places but the version we have is outdated and has some accessibility bugs.

This PR will update to the latest release 1.1.5

### testing
1. publish the search module
2. inspect the source of the module and there is NO aria-expanded attribute
3. apply pr and then npm i
4. inspect the source of the module and there is an aria-expanded=false attribute
5. When the popup appears with search results this changes to aria-expanded=true
